src/__mocks__/taskWrk.test.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import { input, tasks, storage } from './taskWrk';
+import component from './displayHtml';
+import { deleteTrash } from './delete';
+
+describe('Add task functionalities', () => {
+  test('Create a new Object', () => {
+    input();
+    expect(tasks.length).toBe(3);
+  });
+
+  test('Create DOM', () => {
+    document.body.innerHTML = '<ul id="container_ul"></ul>';
+    component();
+    const list = document.querySelectorAll('#container_ul li');
+    expect(list).toHaveLength(3);
+  });
+});
+
```

Comment:
- [ ] Let's add more tests to cover negative scenarios as well. For example test for what happens a new object is not created? we want to the code to anticipate this before it happens